src/main/java/com/lightningrobotics/voidrobot/commands/shooter/ShootCargo.java

```diff
@@ -35,16 +34,20 @@ public ShootCargo(Shooter shooter, Indexer indexer, Turret turret, Vision vision
 	@Override
 	public void execute() {
 
-		var distance = vision.getTargetDistance();
-		var rpm = Constants.DISTANCE_RPM_MAP.get(distance);
-		var hoodAngle = Constants.HOOD_ANGLE_MAP.get(distance);
+		if(vision.hasVision()) {
```

Comment:
This needs to be implemented. Should return true unless NT entry for distance returns -1.